lib/app.rb

```diff
@@ -50,7 +80,31 @@ def handle_add_book
     end
   end
 
-  def save_data
+  def handle_add_game
+    print 'What year was the game published:'
+    publish_date = gets.chomp
+    if publish_date.to_i.zero?
+      puts "\nPlease enter a valid year E.G 2000"
+      return
+    end
+
+    print 'Multiplayer [Y/N]:'
+    multiplayer = gets.chomp.downcase
+
+    print 'When is the game played the last time?:'
+    last_played_at = gets.chomp
```

Comment:
I think it would be better to add some input validation here where you ask the user for the date the game was last played at👍. Currently, if the user enters any random input, the application crashes. Please make sure the user enters an input that `Date.parse()` can work with so they don't break our app😅